.eslintrc.json

```diff
@@ -14,6 +14,7 @@
     "no-shadow": "off",
     "no-param-reassign": "off",
     "eol-last": "off",
+    "camelcase": "off",
     "import/extensions": [

```

Comment:
- [Optional] Dear Team, I noticed the struggle you are going through, After checking the original `.eslintrc.json` config file provided by microverse, I realize you have some extra configuration that introduced an issue to the code base. Please look at [this original configuration file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) and use the content in it for future use.

Reply:
thank you @TadesseDev we would make the correction going forward